ReactWindows/ReactNative.Shared/UIManager/BorderedCanvas.cs

```diff
@@ -40,5 +38,17 @@ public Border Border
                 Children.Insert(0, _border);
             }
         }
+
+#if WINDOWS_UWP
+        /// <inheritdoc />
+        protected override AutomationPeer OnCreateAutomationPeer()
+        {
+            return new DynamicAutomationPeer<BorderedCanvas>(this);
+        }
+#endif
+
+        // TODO: implement runtime change raising event to screen reader #1562
+        /// <inheritdoc />
+        public AccessibilityTrait[] AccessibilityTraits { get; set; }
     }
```

Comment:
I get that it reduces the diff a bit, but should we be implementing the `IAccessible` interface for WPF when it's not being used yet?

Reply:
Moved it in ReactNative project

---
In reply to: [170249922](https://github.com/Microsoft/react-native-windows/pull/1645#discussion_r170249922) [](ancestors = 170249922)